### PR TITLE
Create no_query option

### PR DESCRIPTION
**Pull request by drio18**
Add `--no_query` and `--subfolders` arguments to commands of SubmitCGAP that involve uploading files. 

- `--no_query` : suppresses requests for user input
- `--subfolders`: searches subdirectories of `upload_folder` for files to upload

These facilitate running SubmitCGAP with a job scheduler or in the background of a cloud instance with mounted buckets, as may be desirable for users with files on a local cluster or a cloud service. 

Also, update an import and required `dcicutils` version to latest release, which was created to fix the inability to run SubmitCGAP commands in a python environment lacking `pytest`.

### CODE REVIEW

**Comment by netsettler** [comment]
I'd like to see this called something other than `--remote`. In part this is because everything about `SubmitCGAP` is remote. But also it is because suppressing the query can be done for a variety of reasons. I suggest calling the argument `--no-query` with a short name of `-nq`. I realize that means a truckload of syntactic changes, but I think the result will be more intuitive, easier for users to understand, and clearer for the program control in places.

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "submit_cgap"
-version = "0.8.0"
+version = "0.8.1"
```
Comment (netsettler) [inline pyproject.toml:3]: This PR adds a new feature, so the version should be 0.9.0 under semantic versioning rules.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,22 @@ SubmitCGAP
 Change Log
 ----------
 
+0.8.1
```
Comment (netsettler) [inline CHANGELOG.rst:9]: See comment below in `pyproject.toml`, but this should be 0.9.0.

**Review by netsettler — APPROVED** [review]
This should get a minor version bump, but other than that I see no blocking issues here. My comments are largely informational.

**Review by sbreiff — APPROVED** [review]

**Pull request merged**